python/paddle/tensor/math.py

```diff
@@ -1639,7 +1639,7 @@ def prod(x, axis=None, keepdim=False, dtype=None, name=None):
     Compute the product of tensor elements over the given axis.
 
     Args:
-        x(Tensor): An N-D Tensor, the data type is float32, float64, int32 or int64.
+        x(Tensor): The input tensor, it's data type should be float32, float64, int32, int64.
```

Comment:
Grammar: "it's data" -> "its data"

"it's" means "it is" or "it has"

Reply:
Thanks, done.